src/pretix/base/models/settings.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+
+
+class AdminSetting(models.Model):
+    """
+    An event option is a key-value setting which can be set for an
+    organizer. It will be inherited by the events of this organizer
+    """
+    key = models.CharField(max_length=255)
```

Comment:
How do you feel about using this as the primary key of this model? There is no real reason for an ID.